src/dndbeyond/base/utils.js

```diff
@@ -144,6 +144,91 @@ async function buildAttackRoll(character, attack_source, name, description, prop
     if (damages.length > 0) {
         roll_properties["damages"] = damages;
         roll_properties["damage-types"] = damage_types;
+        
+        if (roll_properties.name === "Chromatic Orb") {
+            const damage_choices = {}
+            for (let dmgtype of ["Acid", "Cold", "Fire", "Lightning", "Poison", "Thunder"]) {
+                let idx = damage_types.findIndex(t => t === dmgtype);
+                damage_choices[roll_properties["damage-types"].splice(idx, 1)[0]] = damages.splice(idx, 1)[0];
```

Comment:
can use `damage_types` here directly instead of `roll_properties`

Reply:
I think that was a holdover from something else...